packages/cli-kit/src/public/node/archiver.ts

```diff
@@ -1,22 +1,49 @@
+/* eslint-disable tsdoc/syntax */
```

Comment:
es lint was complaining about the doc string for `zip`, would love advice on how to make a valid doc string for `zip`'s use case

Reply:
What was the syntax issue? Was the error specific?

Reply:
ty @karreiro that definitely looks cleaner ! with this change im still getting es lint errors 😭 

@matteodepalo this is the error im getting:
![image](https://user-images.githubusercontent.com/18431672/214335397-3e0a6ebf-3ab6-47bf-ad46-293f044a7cd6.png)
 and then when i fix that error i get this one:
 
![image](https://user-images.githubusercontent.com/18431672/214335580-a6291662-60a9-4924-8d08-bc2a2311777f.png)